flight/apps/jetson_comms/message_id.py

```diff
@@ -59,7 +59,9 @@ class Identifiers:
     REQUEST_AOD_LAST_ESTIMATE = 0x51
     REQUEST_ATTITUDE_ESTIMATE = 0x52
     REQUEST_ORBIT_ESTIMATE = 0x53
-    RESOLVE_STATE = 0x54  # Payload simulate forward to current time (prediction)
+    RESOLVE_STATE = (
+        0x54  # Payload simulate forward to current time (prediction)
```

Comment:
Why wrap this single value in parentheses?

Reply:
:) they were part of the previous code from the semester, not this PR. One of the pre-commit tools probably changed it. Funny.